presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchConnectorId.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.elasticsearch;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+public class ElasticsearchConnectorId
```

Comment:
Why this class is needed? Please see: https://github.com/prestodb/presto/pull/11347#pullrequestreview-149464052

Reply:
seems all connectors having ConnectorId, trying to follow them.

Reply:
> seems all connectors having ConnectorId

This is not a good enough reason. Please remove.